### PR TITLE
Don't give out password length in logs

### PR DESCRIPTION
**Pull request by amosonn**
Password length is definitely not the most sensitive piece of information, but it's still better to avoid logging it.

### CODE REVIEW

File: src/dbup-postgresql/PostgresqlExtensions.cs [hunk 1]
```diff
@@ -150,7 +150,7 @@ private static void PostgresqlDatabase(this SupportedDatabasesForEnsureDatabase
         var logMasterConnectionStringBuilder = new NpgsqlConnectionStringBuilder(masterConnectionStringBuilder.ConnectionString);
         if (!string.IsNullOrEmpty(logMasterConnectionStringBuilder.Password))
         {
-            logMasterConnectionStringBuilder.Password = string.Empty.PadRight(masterConnectionStringBuilder.Password.Length, '*');
+            logMasterConnectionStringBuilder.Password = "******";
```
Comment (AdrianJSClark) [inline src/dbup-postgresql/PostgresqlExtensions.cs:153]:
```suggestion
            logMasterConnectionStringBuilder.Password = "******"
```
Small typo on these three lines, which is why the build failed. I'll add suggestions to the others as well, so you can quickly fix them.

**Review by AdrianJSClark — CHANGES_REQUESTED** [review]
Such a simple change. After seeing it I'm not sure why it was done the complicated way to begin with!

If you fix up those extra semi-colons & the build passes I'm happy to merge.

File: src/dbup-sqlserver/SqlServerExtensions.cs [hunk 2]
```diff
@@ -362,7 +362,7 @@ static void GetMasterConnectionStringBuilder(string connectionString, IUpgradeLo
         masterConnectionStringBuilder.InitialCatalog = "master";
         var logMasterConnectionStringBuilder = new SqlConnectionStringBuilder(masterConnectionStringBuilder.ConnectionString)
         {
-            Password = string.Empty.PadRight(masterConnectionStringBuilder.Password.Length, '*')
+            Password = "******";
```
[the remark this answers is not in the document]
Comment (amosonn) [reply src/dbup-sqlserver/SqlServerExtensions.cs]: I'm pretty sure only this one is wrong, the other two are fine. Updated the commit.

**Comment by AdrianJSClark** [comment]
Thanks, @amosonn!

**Pull request merged**